bridge-api/definitions/paged_resources/forward_cursor_scheduled_activity.yml

```diff
@@ -7,7 +7,7 @@ properties:
         items:
             $ref: ../scheduled_activity.yml
     offsetBy:
-        type: number
+        type: string
```

Comment:
Are we rolling these changes together? Or should this be part of a separate change?